examples/analytics.amp.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP #0</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <style amp-custom>
+    amp-img {
+      max-width: 80%;
+      max-height: 80%;
+      width: 100%;
+    }
+  </style>
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+<article>
+  <amp-analytics id="analytics1">
+  {
+    "host": "my-analytics.com",
+    "requests": {
+      "default": "/log?domain=DOMAIN&path=PATH"
```

Comment:
It isn't json-ld, but we can whitelist a new non-script type. I'd vote for
just "json".
On Dec 4, 2015 2:04 PM, "Dima Voytenko" <notifications@github.com> wrote:

> In examples/analytics.amp.html
> <https://github.com/ampproject/amphtml/pull/982#discussion_r46718647>:
>
> > +      max-height: 80%;
> > +      width: 100%;
> > +    }
> > +  </style>
> > +  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
> > +  <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
> > +  <script async src="https://cdn.ampproject.org/v0.js"></script>
> > +</head>
> > +<body>
> > +
> > +<article>
> > +  <amp-analytics id="analytics1">
> > +  {
> > +    "host": "my-analytics.com",
> > +    "requests": {
> > +      "default": "/log?domain=DOMAIN&path=PATH"
>
> I don't see a problem with this to be honest. We could always support both
> forms as well, but the cost of one script tag seems very low to me. Not
> sure what this entail for the validator, but we already use script for
> this purpose for JSON metadata.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ampproject/amphtml/pull/982/files#r46718647>.
>